tests/unit/plugins/python/test_pip.py

```diff
@@ -736,6 +736,18 @@ def test_invalid_json(self):
         self.assertThat(
             str(raised), Contains("Pip packages output isn't valid json"))
 
+    def test_no_json_format(self):
+        self.mock_run.side_effect = [
+            subprocess.CalledProcessError(2, 'pip', 'no such option'),
+            'foo (1.0)',
+        ]
+        self.mock_run.return_value = '[{"name": "foo", "version": "1.0"}]'
```

Comment:
This seems to serve no purpose given the two side_effects. Am I missing something?

Reply:
Indeed it seems I missed a left-over. Thanks for catching it.